modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc

```diff
@@ -1329,37 +1329,69 @@ The following providers are enabled by default:
 - OC\Preview\MP3
 - OC\Preview\TXT
 
-The following providers are disabled by default due to performance or privacy
-concerns:
+The following providers are available, but disabled by default due to performance or privacy concerns:
 
 - OC\Preview\Illustrator
 - OC\Preview\Movie
+- OC\Preview\MSOfficeDoc
 - OC\Preview\MSOffice2003
 - OC\Preview\MSOffice2007
-- OC\Preview\MSOfficeDoc
 - OC\Preview\OpenDocument
+- OC\Preview\StarOffice
 - OC\Preview\PDF
 - OC\Preview\Photoshop
 - OC\Preview\Postscript
-- OC\Preview\StarOffice
 - OC\Preview\SVG
 - OC\Preview\TIFF
 - OC\Preview\Font
 
-The following providers are not available in Microsoft Windows:
+The following providers are only available if either LibreOffice or OpenOffice is installed on the server:
 
-- OC\Preview\Movie
 - OC\Preview\MSOfficeDoc
 - OC\Preview\MSOffice2003
 - OC\Preview\MSOffice2007
 - OC\Preview\OpenDocument
 - OC\Preview\StarOffice
 
+The following providers require the php `imagick` extension:
+
+- OC\Preview\SVG
+- OC\Preview\TIFF
+- OC\Preview\PDF
+- OC\Preview\AI
+- OC\Preview\PSD
+- OC\Preview\EPS
+- OC\Preview\TTF
+- OC\Preview\HEIC
+- OC\Preview\SGI
+
+Install the following additional imagick library when using SVG.
+This library adds imagick support for SVG, WMF, OpenEXR, DjVu and Graphviz:
+
+`sudo apt-get install -y libmagickcore-6.q16-3-extra`
+
+Install the `ffmpeg` when using `OC\Preview\Movie` provider.
+
+`sudo apt install -y ffmpeg`
+
+To get a list of file extensions linked to the image provider, change into the owncloud
```

Comment:
```suggestion
To get a list of file extensions linked to the image provider, change into the ownCloud
```

Reply:
Pls file changes in addition to here also in the referenced core PR.
Then no new `config-to-docs` run is necessary.
Else things get lost on the next run...